spec/models/project_spec.rb

```diff
@@ -42,6 +42,30 @@
       end
     end
 
+    describe "delete associated entities" do
```

Comment:
This is a really handy test! (arguably much handier than the `Shoulda matchers`)

Noticed it's sitting in the `Validations` block, where it should probably be sitting in the `Relationships` block. The shoulda matchers could be updated as well while you are there

`bundle exec rspec spec/models/project_spec.rb --format documentation`

![image](https://user-images.githubusercontent.com/47089130/185606550-9bb8afe5-cd43-48ba-a5ba-58db0685318b.png)

Reply:
I was aiming for it's own block, but I can do Relationships no problem! Thank you!